### PR TITLE
support create schema for openGauss,PostgreSQL

### PR DESCRIPTION
**Pull request by LeeGuoPing**
Refer to #14015 .

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

**Review by strongduanmu — CHANGES_REQUESTED** [review]
Hi @LeeGuoPing , can you take a look for ci error?

**Comment by LeeGuoPing** [comment]
> Hi @LeeGuoPing , can you take a look for ci error?

resolved

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/opengauss/ddl/OpenGaussCreateSchemaStatement.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import lombok.ToString;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.ddl.DDLStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.opengauss.OpenGaussStatement;
+
+/**
+ * OpenGauss create schema statement.
+ */
+@ToString
+public class OpenGaussCreateSchemaStatement extends AbstractSQLStatement implements DDLStatement, OpenGaussStatement {
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/opengauss/ddl/OpenGaussCreateSchemaStatement.java:29]: @LeeGuoPing Please add final for OpenGaussCreateSchemaStatement.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/postgresql/ddl/PostgreSQLCreateSchemaStatement.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import lombok.ToString;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.ddl.DDLStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.postgresql.PostgreSQLStatement;
+
+/**
+ * PostgreSQL create schema statement.
+ */
+@ToString
+public class PostgreSQLCreateSchemaStatement extends AbstractSQLStatement implements DDLStatement, PostgreSQLStatement {
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/postgresql/ddl/PostgreSQLCreateSchemaStatement.java:29]: Same problem.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

**Review by strongduanmu — APPROVED** [review]
@LeeGuoPing Good job.

**Pull request merged**